Develop

tatjama opened the pull request:
Since it is not clear how to display if a user has more than one company (e.g. user mojombo), I decided to show all companies with links.
Also when validating input values, I added separate messages in case the user tries to type special characters or submits a blank input, it seemed to me a better solution than alert

tatjama committed:
HTML Structure

tatjama committed:
CSS Utility Style

tatjama committed:
Create CSS Desktop Style

tatjama committed:
Create Dark Theme

tatjama committed:
Hover on SVG Theme Icons create by filter

tatjama committed:
CSS Style Tablet

tatjama committed:
CSS Style Mobile

tatjama committed:
Add App Behavior with JavaScript

1.Validation Form input ( only letters nad numbers)
2. Prevent submit on empty input
3. Fetch GitHub API
4. Create Defolt OctoCat user on load
5. Error handler
6. Change Color theme on click

tatjama committed:
Apply Correct Color by prefered Color Scheme

tatjama committed:
Fixed issue at user twiter and company links

tatjama committed:
Update Readme.md

tatjama committed:
update readme.md

elvira-quantox commented:
@tatjama Samo par sitnica za ispravku, inače odlično izgleda :)

- Font u search bar-u
- Kada se stranica učita, ako je aktivna light tema, treba odmah da se prikaže link za "dark" temu
- "not available" ne bi trebalo da bude link
- tekst u sekciji "Repos, Followers, Following" bi trebalo centrirati na mobilnom
![2021-10-05_21h15_29](https://user-images.githubusercontent.com/46753272/136088624-719df789-8b2f-425c-a9ea-4db4f1587545.png)

tatjama committed:
Solve issues according Elvira's comment

1. Font in search bar
2. "not available" is not link
3. on mobile device the text at user status section is centered
4. Onload stopped to jump
5. At social section links are centered

tatjama commented:
@elvira-quantox
Hvala :)
Mislim da sam rešila sve ukazane probleme, jedino nisam sigurna za sledeći komentar
> * Kada se stranica učita, ako je aktivna light tema, treba odmah da se prikaže link za "dark" temu
Ne uspevam da reprodukujem takvo ponašanje. Meni se odmah prikaže link za "dark" temu, ali stranica veoma ružno skoči. Ako se komentar odnosio na takvo ponašanje stranice, to sam rešila.

tatjama committed:
Solved issue according to Elvira's comment

tatjama commented:
> > * Kada se stranica učita, ako je aktivna light tema, treba odmah da se prikaže link za "dark" temu
> >   Ne uspevam da reprodukujem takvo ponašanje.

Mislim da sam našla bug i rešila! :)
